headers/types/files/rescue.h

```diff
@@ -0,0 +1,134 @@
+    // [0, 22): The "Common Table" seems to be for easily obtainable items of low value.
+    // Used by TAKE_ITEM_FROM_OUTLAW and DELIVER_ITEM templates, as well as the FIND_ITEM
+    // template with the MISSION_ITEM_COMMON_TABLE subtype. Mostly Seeds, Berries, and Orbs.
+    // Life Seeds, Sitrus Berries, and Reviver Seeds are on this table, for some reason.
+    struct item_id_16 common_table[22];
+    // [22, 42): The "Rare Table" seems to be for higher-value items, more suitable for their
+    // mission types. Used only by the PROSPECT_WITH_CLIENT subtype, as well as the FIND_ITEM
+    // template with the MISSION_ITEM_RARE_TABLE subtype. Consists mostly of Gummis, but Lost Loot
+    // is also present. Sitrus Berries and Reviver Seeds are ALSO on this table, for some reason.
+    struct item_id_16 rare_table[20];
+    // If this is an item table, it's always zero and unused by any template.
+    uint16_t padding[2];
+}
+
+ASSERT_SIZE(struct rescue_item_tables, 176);
+
+// Pokemon tables used by certain mission templates. This could be considered as one table,
+// though barring one exception, none of the ranges overlap.
+// 0xFA0 in rescue.bin
+struct rescue_monster_tables {
+    // [0, 65): Used exclusively for the OUTLAW_NORMAL_2 mission subtype, and consists mostly of
+    // middle-stage evolutions, such as Flaafy and Pidgeotto, with a few of the stronger
+    // fully-evolved pokemon like Masquerain and Beautifly. Seem to be stronger on average than
+    // those in normal_outlaw_3_table.
+    struct monster_id_16 normal_outlaw_2_table[65];
+    // [65, 106): Used exclusively for the OUTLAW_NORMAL_2 mission subtype, and consists mostly of
+    // basic Pokemon, such as Lotad, Teddiursa, and Abra. Also contains all 3 forms of Burmy and
+    // Wormadam.
+    struct monster_id_16 normal_outlaw_3_table[41];
+    // [106, 167): Used exclusively for fleeing outlaws, such as OUTLAW_FLEEING and
+    // TAKE_ITEM_FLEEING_OUTLAW. Consists mostly of cowardly Pokemon such as Magikarp, Dunsparce, or
+    // Luvdisc, as well as fast Pokemon like Doduo, Ponyta, Ninjask, and Yanmega.
+    struct monster_id_16 fleeing_outlaw_table[61];
+    // [167, 211): Used exclusively for the OUTLAW_MONSTER_HOUSE mission subtype, and consists
+    // mostly of evolved Pokemon such as Tyranitar, Nidoking, Honchkrow, and Metagross. Manetric,
+    // Electrike, Chinchou, and Lanturn are also present.
+    struct monster_id_16 monster_house_outlaw_table[44];
+    // [211, 244): Used for the OUTLAW_ESCORT mission subtype, and consists mostly of intimidating
+    // pokemon such as Arbok, Absol, Sharpedo, and Drapion, as well as pseudo-legendaries such as
+    // Metagross, Dragonite, Garchomp, Tyranitar, and Salamence. Every Pokemon on this list is fully
+    // evolved if it has evolutions. This list is the same regardless of Whether Magnezone or
+    // Magnemite accompanies you. Overlaps with the range used by the GUIDE_CLIENT template.
+
```

Comment:
```suggestion
    //
```

just to make it clear it's part of the same comment block